tests/keras/layers/test_convolutional.py

```diff
@@ -14,6 +14,7 @@
 else:
     _convolution_border_modes = ['valid', 'same', 'full']
 
+K.set_image_dim_ordering('tf')
```

Comment:
Most test cases are using input with 'tf' dim_ordering: input_shape=(nb_samples, nb_row, nb_col, stack_size), except for zero_padding since we don't support 'tf' for these layers. Another choice is to dynamically check K.image_dim_ordering and reshape input shape respectively. But this requires changing many test cases.